docs: update AI3 token wallet guides, resolve configs, and add FoxWallet (#962)
autonomys/subspace-docs: docs/ai3_token/wallets/subwallet.mdx

To learn more about SubWallet, visit their official [Website](https://subwallet.app).

---

## Create or Setup Your SubWallet

1. Visit the [SubWallet download page](https://www.subwallet.app/download.html) and install the respective extension version for your browser.
2. Once the extension is installed, open it from your browser's toolbar.
3. The onboarding interface will display several setup options. To initiate a secure staking setup, select **Create a new account**.

:::tip Existing Wallets
If you prefer to import an active pre-existing keyphrase instead of generating a fresh address, select **Import an account**.
:::

![step-3](/img/doc-imgs/subwallet/Subwallet-1.png)

4. Carefully read and accept the SubWallet Terms of Use.

![step-4](/img/doc-imgs/subwallet/Subwallet-2.png)

5. Enter and confirm a strong, master extension password.

![step-5](/img/doc-imgs/subwallet/Subwallet-3.png)

6. Review the structural safety information explaining how to keep your private keys secure. Click on each safety tip to acknowledge that you understand it.

![step-6](/img/doc-imgs/subwallet/Subwallet-4.png)

---

> ## Managing Your Accounts

Reviewer: Remove this section.

To maintain strict accounting precision while staking on the Autonomys Network, it is ideal to generate a fresh target account.

> ### Creating a New Account (Recommended for Staking)

Reviewer: ```suggestion
### Creating a New Account
```

> 1. Provide an easily recognizable local name for your fresh account (e.g., "Autonomys Staking Pool").

Reviewer: ```suggestion
1. Provide an easily recognizable local name for your fresh account.
```

![step-7](/img/doc-imgs/subwallet/Subwallet-5.png)

> 2. Your unique 12-word seed phrase will be generated. Write it down carefully in physical format.

Reviewer: ```suggestion
2. Your unique 12-word seed phrase will be generated. You will need it if you ever forget your password, or decide to transfer the account.
```
